great-idea/index.html

```diff
@@ -1,23 +1,96 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
-  <meta charset="utf-8">
+    <meta charset="utf-8">
 
-  <title>Great Idea!</title>
+    <title>Great Idea!</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
 
-  <!--[if lt IE 9]>
+    <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
-  <body>
+<body>
+
+    <div class="container">
+
+        <header>
+            <nav>
+                <a href="services.html">Services</a>
+                <a href="#">Product</a>
+                <a href="#">Vision</a>
+                <a href="#">Features</a>
+                <a href="#">About</a>
+                <a href="#">Contact</a>
+            </nav>
+            <img class="logo" src="https://tk-assets.lambdaschool.com/c7e8c14d-d7ad-4d21-9c18-4a68c77e5ad5_logo.png" alt="Great Idea! Company logo.">
+        </header>
+
+        <section class="cta">
+            <div class="cta-text">
+                <h1>Innovation<br> On<br> Demand</h1>
```

Comment:
The page is looking spot on! The only thing you need to change is the positioning of the words in the h1 to better reflect the design. Multiple ways to go about it, such as separating the words into their own elements to make positioning with flex easier, or you can try manipulating the width, padding, etc. Feel free to change the code, as long as you get the intended effect.